tracer/test/Datadog.Trace.ClrProfiler.Managed.Tests/AutoInstrumentation/Proxy/AwsApiGatewaySpanFactoryTests.cs

```diff
@@ -0,0 +1,50 @@
+// <copyright file="AwsApiGatewaySpanFactoryTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using Datadog.Trace.ClrProfiler.AutoInstrumentation.Proxy;
+using FluentAssertions;
+using Xunit;
+
+namespace Datadog.Trace.ClrProfiler.Managed.Tests.AutoInstrumentation.Proxy;
+
+public class AwsApiGatewaySpanFactoryTests
+{
+    private readonly AwsApiGatewaySpanFactory _factory;
+    private readonly Tracer _tracer; // this is a mocked instance of the tracer
+
+    public AwsApiGatewaySpanFactoryTests()
+    {
+        _factory = new AwsApiGatewaySpanFactory();
+        _tracer = ProxyTestHelpers.GetMockTracer();
+    }
+
+    [Fact]
+    public void CreateSpan_CreatesSpanWithCorrectProperties()
+    {
+        var startTime = DateTimeOffset.UtcNow;
+        var data = new InferredProxyData("aws-apigateway", startTime, "test.api.com", "GET", "/api/test", "prod");
+
+        var scope = _factory.CreateSpan(_tracer, data);
+
+        scope.Should().NotBeNull();
+        var span = scope!.Span;
+        span.Should().NotBeNull();
+        span.OperationName.Should().Be("aws.api-gateway");
```

Comment:
The span name in the Python, Go, and JS implementation is `aws.apigateway`. Any way to change this to be consistent?

Reply:
```suggestion
        span.OperationName.Should().Be("aws.apigateway");
```

Left a comment where it comes from as well, as this is just a test. Unsure how I got that `-` in there 😅

Reply:
I think that the .NET implementation should adhere to that with the above suggestion applied here and in the operation name as well